.github/workflows/mpl-master-test.yml

```diff
@@ -14,13 +14,13 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ["3.9.x"]
+        python-version: ["3.x"]
```

Comment:
You might have had a reason to keep this at `3.9` but `matplotlib` is very good about keeping up with CPython releases.